lib/JSON/CalDAV/CalendarHandler.php

```
@@ -0,0 +1,476 @@
+        }
+
+        if ($withRights) {
+            if ($node->getACL()) {
+                $subscription['acl'] = $node->getACL();
+            }
+        }
+
+        return $subscription;
+    }
+
+    public function updateSharees($path, $node, $jsonData) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$node'. (undefined)

(UnusedFormalParameter)